.github/workflows/Stale.yml

```diff
@@ -0,0 +1,93 @@
+# This workflow warns and then closes issues and PRs that have had no activity for a specified amount of time.
+#
+# Copyright (c) Microsoft Corporation. All rights reserved.

```

Comment:
Is this the correct copyright?

Reply:
Honestly, guidance is somewhat conflicting (or at least not super clear). The generic source code copyright is documented as follows:

```
All checked in source files and sample code files should have one of the source code notices below included in a comment at the top of the file. Use the MS Public License notice only if the code sample or file will be released under the Microsoft Public License.

Standard notice for source code

// Copyright (c) Microsoft Corporation. All rights reserved.
```

Not posting links here since they're useless to others in a public project.

However, I still see the option with "All rights reserved." somewhere else. I think I'll drop "All rights reserved." from here for consistency with edk2 contributions (and most other open source contributions) if no further feedback.

Reply:
Updated in PR commit https://github.com/microsoft/mu_devops/pull/27/commits/7fb1e6ec615e04f8b010c9f7043028bd254c5939